src/main/java/OGF.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Scanner;
+public class OGF {
+    private static void printBreakLine(){
+        System.out.println(("____________________________________________________________"));
+    }
+
+    private static void printTaskAdded(Task task, int taskIndex){
+        System.out.println("Alright, adding this task to the list: ");
+        System.out.println(task);
+        System.out.printf("You have %d tasks in the list.%n", taskIndex+1);
+        printBreakLine();
+    }
+    public static void main(String[] args) {
+        String logo = "\n" +
+                " _____  _    _                   _____ ______ \n" +
+                "|  _  || |  | |                 |  __ \\|  ___|\n" +
+                "| | | || |_ | |__    ___  _ __  | |  \\/| |_   \n" +
+                "| | | || __|| '_ \\  / _ \\| '__| | | __ |  _|  \n" +
+                "\\ \\_/ /| |_ | | | ||  __/| |    | |_\\ \\| |    \n" +
+                " \\___/  \\__||_| |_| \\___||_|     \\____/\\_|    \n" +
+                "                                              \n" +
+                "                                              \n";
+        System.out.println("Welcome! I'm your \n" + logo + "Nice to meet you!");
+        System.out.println("What can I do for you?");
+
+
+        Task[] tasks = new Task[100];
+        int numItem = 0;
+        Scanner scanner = new Scanner(System.in);
+        int taskNo;
+        while (true) {
+            String input = scanner.nextLine();
+            switch (input.split(" ")[0]) {
+                case ("bye"):
+                    System.out.println("Bye bye now!");
+                    return;
+                case ("list"):
+                    System.out.println("Here are your tasks for today: ");
+                    for (int i = 0; i < numItem; i++){
+                        System.out.print(i+1 + ". ");
+                        System.out.println(tasks[i]);
+                    }
+                    printBreakLine();
+                    break;
+                case ("mark"):
+                //Fallthrough
+                case ("unmark"):
+                    taskNo = Integer.parseInt(input.split(" ")[1])-1;
+
+                    if (input.split(" ")[0].equals("mark")){
```

Comment:
perhaps set your mark case as its own case rather than a fallthrough and processing it within your unmark case. helps with readability as well